src/components/ListItem.spec.js

```diff
@@ -0,0 +1,31 @@
+import { mount, shallowMount } from '@vue/test-utils';
+import ListItem from './ListItem';
+
+const mountItem = shallowMount(ListItem, {
+    propsData: {
+        item: {
+            text: "original"
+        }
+    }
+});
+
+describe("ListItem", () => {
+    test('go in edit mode', () =>{
```

Comment:
```suggestion
    test('click on edit button shows input field', () =>{
```
I like to put that into a `describe` block:
```js
describe('click on edit button', () => {
  beforeEach(() => mountItem.find("#edit").trigger("click"))

  it('shows input field', () => {
    expect(mountItem.contains("input")).toBe(true)
  })
})